test/browser/browser.sh

```diff
@@ -1,41 +1,24 @@
 #!/bin/sh
-set -eux
-
-# like "basic", passed on to run-test.sh
-PLAN="$1"
```

Comment:
This needs to be put back. We need to tell the run-tests.sh in the container about which test plan to run.